src/compiler.jl

```diff
@@ -21,7 +21,7 @@ include(joinpath("compiler", "driver.jl"))
 
 function __init_compiler__()
     # enable generation of FMA instructions to mimic behavior of nvcc
-    LLVM.clopts("--nvptx-fma-level=1")
+    LLVM.clopts("-nvptx-fma-level=1")
```

Comment:
Not sure about this one, but it struck me as odd since all LLVM clopts only use one dash.
And the one test file has: `3:; RUN: llc < %s -march=nvptx64 -mcpu=sm_20 -nvptx-fma-level=1 | FileCheck %s -check-prefix=FMA`